official/resnet/resnet.py

```diff
@@ -516,6 +572,10 @@ def __init__(self, resnet_size_choices=None):
         '--data_dir', type=str, default='/tmp/resnet_data',
         help='The directory where the input data is stored.')
 
+    self.add_argument(
+        '--parallel_calls', type=int, default=5,
```

Comment:
How about `input_parallel_calls`, so that it's clear that this flag only applies to the # of parallel calls during the input pipeline (and that it's not related to the multi-gpu functionality)

Reply:
Looking at the new data processing guide ( https://www.tensorflow.org/versions/master/performance/datasets_performance#parallelize_data_extraction ), I ended up going with num_parallel_calls to be consistent with that. I think with the greater detail in the help comment, it should be pretty clear that it's specifically for input processing?